drivers/mfd/Kconfig.axp192

```diff
@@ -4,7 +4,7 @@
 config MFD_AXP192
 	bool "AXP192 PMIC multi-function device driver"
```

Comment:
please also align Kconfig names/prompts, otherwise it may be misleading

Reply:
Do you prefer the suffix  _AXP192_AXP2101 or _AXP192_2101 for the Kconfig names?